unit-tests/live/aus/test-D415.py

```diff
@@ -0,0 +1,56 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D415
+
+import pyrealsense2 as rs
+from rspy import devices, log, test, file, repo
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Init colorizer test")
+try:
+
+    pipeline = rs.pipeline()
+    pipeline.start()
+    for i in range(10):
+        frames = pipeline.wait_for_frames()
+except:
+    test.unexpected_exception()
+test.finish()
+
+#############################################################################################
+
+test.start("Test counters names")
+try:
+    counters_list = rs.aus_get_counters_list()
+    expected_counters_dict = \
+        {
+            'RS2_AUS_INTEL_REALSENSE_D415_COLOR_TIMER': -1,
+            'RS2_AUS_CONNECTED_DEVICES_COUNTER': 1,
+            'RS2_AUS_DS5_CONNECTED_DEVICES_COUNTER': 1,
+            'RS2_AUS_INTEL_REALSENSE_D415_DEPTH_TIMER': -1.
+        }
+    test.check_equal_lists( counters_list, expected_counters_dict.keys())
+except:
+    test.unexpected_exception()
+test.finish()
+
+#############################################################################################
+
+test.start("Test counters values")
+try:

```

Comment:
No need to try / except on LibCI tests, LibCI will fail the test on exception.
You only need to use it when you want to catch an exception yourself, if you expect it.